presenters/result_test.go

```diff
@@ -54,4 +54,44 @@ var _ = Describe("Result", func() {
 			})
 		})
 	})
+
+	Describe("#GetResultPresenter", func() {
+		Context("given a valid result", func() {
+			It("returns a result presenter", func() {
+				user := FabricateUser(faker.Email(), faker.Password())
+				result := FabricateResult(user)
+				adLink1 := FabricateAdLinkWithParams(result, models.AdLinkPositionTop)
+				adLink2 := FabricateAdLinkWithParams(result, models.AdLinkPositionBottom)
+				link := FabricateLink(result)
+				expectedAdLinkCollection := map[string][]string{
+					models.AdLinkPositionTop: {adLink1.Link},
+					models.AdLinkPositionBottom: {adLink2.Link},
+					models.AdLinkPositionSide: {},
+				}
+				result, err := models.GetResultByIdWithRelations(result.Id)
+				if err != nil {
+					Fail("Failed to get result by ID")
+				}
+
+				presenter := presenters.GetResultPresenter(result)
+				Expect(presenter.Id).To(Equal(result.Id))
```

Comment:
```suggestion

				Expect(presenter.Id).To(Equal(result.Id))
```

Reply:
fixed in [d183b2f](https://github.com/carryall/go-google-scraper-challenge/pull/73/commits/d183b2f3da22e3829620a9d690c55a3ece983ff6)